2024/ig-security.html

```diff
@@ -186,6 +186,24 @@ <h2>
                 In joint with W3C's <a href="https://www.w3.org/2001/tag/">Technical Architecture Group (TAG)</a> and <a href="https://www.w3.org/groups/ig/privacy/">PING</a>, with a specific focus on Security aspect.
               </p>
             </dd>
+            <dt id="TM" class="spec">
+              A Threat Model for the Web
+            </dt>
+            <dd>
+              <p>
+                A description of the Web's threat model, which new specifications need to maintain.
+                This threat model may include goals that have not yet been achieved across the whole
+                web platform, but which will still be enforced in reviews of new and changed
+                specifications. This will be developed with input from other relevant groups such as
+                the TAG, PING, WebAppSec, and the <a
+                href="https://www.w3.org/groups/cg/tmcg/">Threat Modeling Community Group</a>.
+              </p>
+              <p><b>Potential input documents:</b></p>
+              <ul>
+                <li><a href="https://chromium.googlesource.com/chromium/src/+/master/docs/security/web-platform-security-guidelines.md">Chromium Web Platform Security guidelines</a></li>
+                <li>The <a href="https://xsleaks.dev/">XS-Leaks Wiki</a></li>
```

Comment:
Ideally the charter would list a few more input documents from outside of Chromium's orbit. These are just the documents I had handy.